models/src/test/java/com/rudderstack/models/RudderServerConfigParseTest.kt

```diff
@@ -468,19 +65,106 @@ open class RudderServerConfigParseTest {
             ),
         )
     }
-}
-class RudderServerConfigParseTestJackson : RudderServerConfigParseTest() {
-    init {
-        jsonAdapter = JacksonAdapter()
+
+    @Test
+    fun testObjectSerializationDeserialization() {
```

Comment:
💡 Just a minor change here for better understanding of the methods goal:
```

@Test
fun `given a  rudderServerConfig object, when serializing, then the operation completes successfully`() {
```
I think this can be a bit more descriptive. 😀